BGL/include/CGAL/boost/graph/IO/STL.h

```diff
@@ -267,11 +271,11 @@ bool write_STL(std::ostream& os,
     for(const face_descriptor f : faces(g))
     {
       const halfedge_descriptor h = halfedge(f, g);
-      Point_ref p = get(vpm, target(h, g));
-      Point_ref q = get(vpm, target(next(h, g), g));
-      Point_ref r = get(vpm, source(h, g));
+      const Point_ref p = get(vpm, target(h, g));
```

Comment:
`vpm's reference might already provide a const type. This will lead to compilation issues in such a case.